tests/integration/conftest.py

```diff
@@ -80,7 +79,6 @@ def conf(sbwshome_dir):
 
 @pytest.fixture(scope='session')
 def sbwshome(sbwshome_dir, args, conf):
-    init.main(args, conf)
     return sbwshome_dir
```

Comment:
This function looks unnecessary. Instead of using this fixture, why not just use the parent `sbwshome_dir` fixture in tests?